apps/keyboard/js/settings/word_list_converter.js

```diff
@@ -405,7 +407,7 @@ TSTBlobBuilder.prototype.toBlobArray = function() {
     this._emitNode(node);
   }, this);
 
-  return this._output;
+  return this._outputBuffer;
```

Comment:
I wonder if `this._output.buffer` is a better pointer to the buffer instance instead of yet another private property? But that's not really important.